addons/ofxOsc/src/ofxOscSender.cpp

```diff
@@ -47,12 +47,16 @@ ofxOscSender::~ofxOscSender()
 		shutdown();
 }
 
-void ofxOscSender::setup( std::string hostname, int port )
+void ofxOscSender::setup( std::string hostname, int port, bool enableBroadcast )
```

Comment:
also, the broadcast changes appear here - was there some mixup with the PR branch root?